homeassistant/components/raspberrypi/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""The Raspberry Pi integration."""
+from __future__ import annotations
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up Raspberry Pi."""
+
+    await hass.config_entries.flow.async_init(
+        "rpi_power", context={"source": "onboarding"}
```

Comment:
This integration needs to be moved into `raspberrypi`, not live standalone.

Reply:
Yes, but in a follow-up